src/apps/chatter/room/index.js

```diff
@@ -10,6 +10,8 @@ import LoadingIndicator from '@app-elements/loading-indicator'
 import { Link } from '@app-elements/router'
 import { useRequest } from '@app-elements/use-request'
 import { ChatBox } from '/elements/chatbox'
+import { WebSocketServiceInstance } from '/services/WebSocketService'
```

Comment:
We try to follow the folder naming that matches npm, so all lowercase, kebab: `websocket-service`

Reply:
@bkmaibach the new util should follow kebab file naming as well.

Reply:
Ten four!